### PR TITLE
Feat/define logo and colors

### PR DESCRIPTION
**Pull request by tiznajo**
### Description

**TL;DR:** This PR incorporates changes related to the update of the application logo, color theme, favicon, fixes, and the addition of a Terms and Conditions page for service providers.

**Logo and Favicon Update:**
Updated the BeTokyo application logo and favicon. The new logo is designed to better represent the brand's identity. The favicon has been updated for consistency with the new logo. 

**Color Theme Update:**
The application's color theme has been updated to improve user experience, especially in dark mode. The new color scheme improves the visibility and readability of the text and UI elements. The primary color for dark mode has been changed to a lighter hue for better visibility.

**Fixes:**
A series of fixes have been made to improve the functionality and stability of the application. These include bug fixes, code optimizations, and improvements in load time.

**Terms and Conditions Page for Service Providers:**
A new Terms and Conditions page has been created specifically for service providers. This page lays out the terms of use for service providers using the BeTokyo application for managing reservations. It is designed to provide clarity on the roles, responsibilities, and expectations for service providers using the platform.

All these changes aim to enhance the user experience for both customers and service providers using the BeTokyo application, by providing a more intuitive interface and clearer information.

### User Stories/Bugs affected

Related #1

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -37,88 +41,91 @@ const config = {
...
           {
-            position: 'right',
-            label: 'Tarifas',
-            href: '/pricing'
+            position: "right",
+            label: "Características",
+            href: "/features",
           },
           {
+ feat/define_logo_and_colors
```
Comment (aseguragonzalez) [inline docusaurus.config.js:58]: Remove this line

**Review by aseguragonzalez — CHANGES_REQUESTED** [review]
